README.md

```diff
@@ -2,7 +2,7 @@ Kateheo.ru Distribution
 ========
 
 CircleCI integration status:
-[![Circle CI](https://circleci.com/gh/sfi-ru/KateheoDistr/tree/master.svg?style=svg)](https://circleci.com/gh/sfi-ru/KateheoDistr/tree/master)
+[![Circle CI](https://circleci.com/gh/psmb/KateheoDistr/tree/master.svg?style=svg)](https://circleci.com/gh/psmb/KateheoDistr/tree/master)
 
 ##Installation
 
```

Comment:
You may just link to our contributor page: no need to maintain multiple installation guides in different places... http://psmb.github.io/